### PR TITLE
Updated Readme to include pph_unlock_password_data update & account c…

### PR DESCRIPTION
**Pull request by dhishan**
Updated Readme to include pph_unlock_password_data update & account creation in locked phase

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,10 +216,13 @@ in disk.
   username_lengths[0] = strlen("Alice");
   username_lengths[1] = strlen("bob");
 
+  unsigned int *password_lengths = malloc(sizeof(*password_lengths)*2);
```
Comment (SantiagoTorres) [inline README.md]: We should free password lengths somewhere later.

**Comment by SantiagoTorres** [comment]
Hi @dhishan. Thanks for your PR! I only found that nit.

File: README.md (hunk 1 above)
Comment (dhishan) [inline README.md]: Totally. I completely forgot. I will update that too in the Readme. 
Thanks :)

**Comment by dhishan** [comment]
Also I have created a sample application, separating the creation,
reloading and unlocking with exploring various functions of the library.
Will it add value if we add it as an sample application in the repo?

File: README.md [hunk 2]
```diff
@@ -230,6 +241,12 @@ in disk.
   pph_store_context(context,"securepasswords");
   pph_destroy_context(context);
 
+  free(password_lengths);
+  free(usernames);
+  free(passwords);
+  free(username_lengths);
```
Comment (SantiagoTorres) [inline README.md]: Thanks for checking the rest! 

Could you get read of the additional whitespace please? :)

**Comment by SantiagoTorres** [comment]
👍 thanks!

**Pull request merged**